velox/core/PlanNode.h

```diff
@@ -1741,6 +1741,15 @@ class OrderByNode : public PlanNode {
         sortingKeys.size(),
         sortingOrders.size(),
         "Number of sorting keys and sorting orders in OrderBy must be the same");
+    std::unordered_set<std::string> sortingKeySet;
+    // Reject duplicated sorting keys.
```

Comment:
1. duplicated -> duplicate
2. Move this comment before 'sortingKeySet' variable.